packages/components/src/components/wizard/wizard.tsx

```diff
@@ -1,42 +1,56 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
-import Step from './wizard-step.jsx';
+import Step from './wizard-step';
+
+type TWizard = {
+    className?: string;
+    initial_step: number;
+    onStepChange: (prop: { [key: string]: number }) => void;
+    nav: React.ReactNode;
+    selected_step_ref: () => React.MutableRefObject<HTMLElement>;
+};
 
-const Wizard = ({ children, className, initial_step, onStepChange, nav, selected_step_ref }) => {
+const Wizard = ({
+    children = [],
+    className,
+    initial_step = 1,
+    onStepChange = () => undefined,
```

Comment:
looks a liitle bit strange, maybe we can better to check whether the function is passed and the call it on the 36 line, then create default empty function?

Reply:
Will refactor this